src/sidebar/annotation-service.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+/**
+ * Return the annotation service which the client should retrieve annotations from
```

Comment:
I think it would be slightly clearer to say that it returns the _configuration_ for the annotation service.